.github/workflows/test.yml

```diff
@@ -0,0 +1,42 @@
+---
+name: Molecule Test
+
+on: [push]
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: molecule
+        uses: robertdebock/molecule-action@2.0.0
+        with:
+          entrypoint: /usr/local/bin/molecule
+          args: lint
+  test:
+    needs:
+      - lint
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - image: "centos"
+            tag: "6"
+          - image: "centos"
+            tag: "7"
+          - image: "centos"
+            tag: "8"
+          - image: "amazonlinux"
+            tag: "2"
```

Comment:
Normally this would be in the Molecule files but having defined here might be nice when the GH actions can alert you what combo is failing rather than all of them